packages/bpk-mixins/_index.scss

```diff
@@ -18,6 +18,7 @@
 
 @import 'src/bonds.scss';
 @import 'src/mixins/badges.scss';
+@import 'src/mixins/badges-v2.scss';
```

Comment:
Instead of doing this here because we are going for the V2 component we can just do this import inside the `BpkBadge.module.scss`